Crash on "Message in Console" pane

The whole client crashes if a message is typed in the *console* pane. 

```
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x1322740]

goroutine 1 [running]:
github.com/rivo/tview.(*Application).Run.func1(0xc0001b8000)
	/Users/mit/go/src/github.com/rivo/tview/application.go:149 +0x82
panic(0x138d7e0, 0x16e85f0)
	/usr/local/opt/go/libexec/src/runtime/panic.go:522 +0x1b5
github.com/thorsager/t-slacker/connection.(*Connection).SendMessage(0x0, 0x13fce24, 0x8, 0xc000685b28, 0x4, 0x23, 0x0)
	/Users/mit/go/src/github.com/thorsager/t-slacker/connection/slack.go:150 +0x40
github.com/thorsager/t-slacker/runtime.(*AppRuntime).onInput(0xc0001b8080, 0xc0001be000, 0xc000685b28, 0x4)
	/Users/mit/go/src/github.com/thorsager/t-slacker/runtime/app_context.go:301 +0x182
github.com/thorsager/t-slacker/pane.newPane.func1(0xd)
	/Users/mit/go/src/github.com/thorsager/t-slacker/pane/pane.go:57 +0x68
github.com/rivo/tview.(*InputField).InputHandler.func1.9(0xc00000000d)
	/Users/mit/go/src/github.com/rivo/tview/inputfield.go:486 +0x66
github.com/rivo/tview.(*InputField).InputHandler.func1(0xc00042a680, 0xc0004892b0)
	/Users/mit/go/src/github.com/rivo/tview/inputfield.go:564 +0x3a3
github.com/rivo/tview.(*Box).WrapInputHandler.func1(0xc00042a680, 0xc0004892b0)
	/Users/mit/go/src/github.com/rivo/tview/box.go:160 +0x5d
github.com/rivo/tview.(*Application).Run(0xc0001b8000, 0x0, 0x0)
	/Users/mit/go/src/github.com/rivo/tview/application.go:234 +0x396
github.com/thorsager/t-slacker/runtime.(*AppRuntime).Run(0xc0001b8080, 0x15, 0xc00001a140)
	/Users/mit/go/src/github.com/thorsager/t-slacker/runtime/app_context.go:237 +0xa7
main.main()
	/Users/mit/go/src/github.com/thorsager/t-slacker/main.go:37 +0x17c
```

--- a/runtime/app_context.go
+++ b/runtime/app_context.go
@@ -294,9 +294,11 @@ func (c *AppRuntime) onInput(p *pane.Pane, input string) {
 		}
 	} else {
 		team := c.GetTeam(p.TeamId)
-		err := team.SendMessage(p.ChannelId, input)
-		if err != nil {
-			p.Logf("ERROR", "unable to send message at this time: %v", err)
+		if team != nil { // should only be nil on console-pane
+			err := team.SendMessage(p.ChannelId, input)
+			if err != nil {
+				p.Logf("ERROR", "unable to send message at this time: %v", err)
+			}
 		}
 	}
 }

